packages/web/app/components/Field/DateField.jsx

```diff
@@ -88,37 +112,54 @@ export const DateInput = ({
       setIsPlaceholder(false);
       setCurrentText(formattedValue);
       if (outputValue === 'Invalid date') {
-        onChange({ target: { value: '', name } });
+        clearValue();
         return;
       }
 
       onChange({ target: { value: outputValue, name } });
     },
-    [onChange, format, name, min, max, saveDateAsString, type],
+    [onChange, format, name, saveDateAsString, type, clearValue],
   );
 
+  const onKeyDown = event => {
+    if (event.key === 'Backspace') {
+      clearValue();
+    }
+    // if the user has started typing a date, turn off placeholder styling
+    if (event.key.length === 1) {
+      setIsPlaceholder(false);
+    }
```

Comment:
Im not sure this important but mabye worth not setting state unnecisary.
```suggestion
    if (event.key.length === 1 && isPlaceholder) {
      setIsPlaceholder(false);
    }
```

Reply:
Yep true